internal/confmapprovider/discovery/bundle/bundle.d/receivers/smartagent-collectd-mysql.discovery.yaml.tmpl

```diff
@@ -1,5 +1,5 @@
 {{ receiver "smartagent/collectd/mysql" }}:
-  enabled: true
+  enabled: false
```

Comment:
As discussed offline, we should keep this enabled until the ootb dashboards are updated (w/ mysql disabled)